src/Citeproc/Eval.hs

```diff
@@ -1542,10 +1547,10 @@ eText (TextVariable varForm v) = do
                       coerce (referenceId ref)
             return NullOutput
 
-    "DOI"   -> handleIdent IdentDOI
-    "PMCID" -> handleIdent IdentPMCID
-    "PMID"  -> handleIdent IdentPMID
-    "URL"   -> handleIdent IdentURL
+    "DOI"   -> handleIdent fixShortDOI IdentDOI
```

Comment:
Passing `fixShortDOI` here so that the link anchor for a short DOI like `10/abcde` appears as `abcde` instead.

Reply:
Is this the desired behavior?  Where is this documented?  Does pandoc currently do this or is it a behavior change?

Reply:
I believe this is how pandoc currently works.  I had to squint at [`linkifyVariables`](https://github.com/jgm/pandoc/blob/82638ad53bb47acde835083390f8211ac21cda6c/src/Text/Pandoc/Citeproc.hs#L541-L547) for a while earlier to realize it (note `fixShortDOI x` is passed to the `tolink` function, which determines the link anchor/target).  I also just confirmed this behavior with the release copy of pandoc. 

As for whether this is desirable:  I'd say yes, since every shortDOI `10/abcde` corresponds to the real DOI `abcde`, right? 
 If we don't do it this way, and we have a `<text variable="doi" prefix="https://doi.org/".`, the link will render incorrectly as `https://doi.org/10/abcde` when given a shortDOI `10/abcde`.  Instead, it should be `https://doi.org/abcde`.

Reply:
OK, right.